Makefile

```diff
@@ -24,6 +24,9 @@ clean:
 client: generate
 	hack/build-client.sh 2020-04-30 2021-01-31-preview
 
+deploy:
```

Comment:
add a note to say that this is hacky and hopefully temporary and open for improvement